modules/services/mbsync.nix

```diff
@@ -0,0 +1,95 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.mbsync;
+
+  mbsyncOptions =
+    [ "--all"
+    ] ++ optional (cfg.configFile != null) "--config ${cfg.configFile}";
+in
+{
+  meta.maintainers = [ maintainers.pjones ];
+
+  options.services.mbsync = {
+    enable = mkEnableOption "mbsync";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.isync;
+      defaultText = "pkgs.isync";
+      example = literalExample "pkgs.isync";
+      description = "The package to use for the mbsync binary.";
+    };
+
+    frequency = mkOption {
+      type = types.str;
+      default = "*:0/5";
+      description = ''
+        How often to run mbsync.  This value is passed to the systemd
+        timer configuration as the onCalendar option.  See
+        systemd.time(7) for more information about the format.
```

Comment:
You can refer to man pages using a specific docbook markup (which admittedly is a bit clunky):
```xml
timer configuration as the onCalendar option.  See
systemd.time(7) for more information about the format.
```
becomes
```xml
timer configuration as the onCalendar option.  See
<citerefentry>
  <refentrytitle>systemd.time</refentrytitle>
  <manvolnum>7</manvolnum>
</citerefentry>
for more information about the format.
```

Reply:
Thanks for the tips.  Fixed.